sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/repository/support/CosmosEntityInformation.java

```diff
@@ -64,6 +65,15 @@ public class CosmosEntityInformation<T, ID> extends AbstractEntityInformation<T,
     private final Field id;
     private final Field partitionKeyField;
     private final Field versionField;
+
+    /**
+     * @return fields with @Transient annotation
+     */
+    public List<String> getTransientFields() {
```

Comment:
Can we move this down with the other getters instead of up here in the variable definitions?

Reply:
Yep, have done